<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HricsWidget</class>
 <widget class="QWidget" name="HricsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>991</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-76</y>
        <width>551</width>
        <height>1065</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_24">
       <item>
        <widget class="QCheckBox" name="enableHri_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>Enable Human Robot Interactions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAStarInGrid">
         <property name="text">
          <string>A* In Grid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_11" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonInitHRICS">
            <property name="text">
             <string>Init HRICS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxPosOr">
            <property name="text">
             <string>Position / Orientation</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxDraw">
         <property name="title">
          <string>Draw</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxDrawDistance">
            <property name="text">
             <string>Distance</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxDrawGaze">
            <property name="text">
             <string>Gaze</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxDrawBox">
            <property name="text">
             <string>Box</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBoxDrawGrid">
            <property name="text">
             <string>Grid</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxDrawVectorField">
            <property name="text">
             <string>Vector Field</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBoxEntireGrid">
            <property name="text">
             <string>Entire Grid</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBoxDrawRandPoints">
            <property name="text">
             <string>RandPts</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBoxDrawHumanColorFromConf">
            <property name="text">
             <string>Hum. Color From Conf.</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBoxDrawReachable">
            <property name="text">
             <string>Reachable</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxMisc">
         <property name="title">
          <string>Misc</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBoxHriPathDistance">
            <property name="text">
             <string>Path Distance</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxHRINoRobot">
            <property name="text">
             <string>No Robot</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxAutoLoadGrid">
            <property name="text">
             <string>Auto Load Grid</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBoxHriPlannerTRRT">
            <property name="text">
             <string>Planner T-RRT</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBoxCameraBehindHuman">
            <property name="text">
             <string>Camera Behind Hum.</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxRecomputeCost">
            <property name="text">
             <string>Recom. Cost In Grid</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBoxComputeOTP">
            <property name="text">
             <string>Compute OTP</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxHRICSPlanner">
         <property name="title">
          <string>HRICS Planner</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QCheckBox" name="checkBoxHRICS_MOPL">
              <property name="text">
               <string>HRICS WS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonInitGrids">
              <property name="text">
               <string>Init Grids</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonDeleteGrids">
              <property name="text">
               <string>Delete Grids</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButtonDeleteGrid">
               <property name="text">
                <string>Delete Planner</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButtonComputeAllCellCost">
               <property name="text">
                <string>(Re)Compute All Cell Cost</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButtonVectorField">
               <property name="text">
                <string>Compute Vector Field</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButtonLoad">
               <property name="text">
                <string>Load</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButtonSave">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="pushButtonComputeOtpConfig">
               <property name="text">
                <string>Compute Otp Config</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxCostFunctions">
         <property name="title">
          <string>Cost Functions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButton">
               <property name="text">
                <string>Ball Dist</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_2">
               <property name="text">
                <string>Box Dist</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="whichTestBox">
            <item>
             <property name="text">
              <string>Distance</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Visibility</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Naturality</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Reachability</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Combined</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QGridLayout" name="gridLayout_17">
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item row="1" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Visibility</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxVisibility">
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSlider" name="horizontalSliderVisibility">
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Distance</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxDistance">
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSlider" name="horizontalSliderDistance">
               <property name="maximum">
                <number>500</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Reachable</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxReachable">
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QSlider" name="horizontalSliderReachable">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Natural</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxNatural">
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QSlider" name="horizontalSliderNatural">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Grouping</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
             </item>
             <item row="6" column="2">
              <widget class="QSlider" name="horizontalSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxGrids">
         <property name="title">
          <string>Grids</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QRadioButton" name="radioButtonXline">
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonYline">
               <property name="text">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonZline">
               <property name="text">
                <string>Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxDrawOnlyOneLine">
               <property name="text">
                <string>Draw Only One Slice</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_14" native="true">
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Slice number</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QSlider" name="horizontalSliderWhichGridLine">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxWhichGridLine">
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="maximum">
                <double>500.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_13" native="true">
            <layout class="QGridLayout" name="gridLayout_35">
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item row="2" column="0">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Color1</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Color2</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QSlider" name="horizontalSliderColor1">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QSlider" name="horizontalSliderColor2">
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxColor1">
               <property name="maximum">
                <double>2.500000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxColor2">
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="maximum">
                <double>2.500000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Zone Size</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxZoneSize">
               <property name="maximum">
                <double>2.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QSlider" name="horizontalSliderZoneSize">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Cell Size</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxCellSize">
               <property name="maximum">
                <double>10.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QSlider" name="horizontalSliderCellSize">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>PlanCell Size</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxPlanCellSize"/>
             </item>
             <item row="6" column="2">
              <widget class="QSlider" name="horizontalSliderPlanCellSize">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
